rakefile.netstd.rb

```diff
@@ -106,9 +106,17 @@
 end
 
 desc "Restore NuGet packages"
-exec :restore do |cmd|
-  cmd.command = nuget_command
-  cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+task :restore do
+  restore_cmd = Exec.new
+  restore_cmd.command = nuget_command
+  restore_cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+  restore_cmd.execute
+
+  # performing restore on the solution doesn't restore

```

Comment:
but I'm inclined to go with it. It may get better when the tooling's done. Or we'll learn stuff later.